### PR TITLE
Dependabot + New Year + Test fixes

### PR DESCRIPTION
**Pull request by lluiscampos**
Supersedes #280.

### CODE REVIEW

File: modules/testscfg.sh [hunk 1]
```diff
@@ -0,0 +1,47 @@
...
+  touch work/testscfg.tmp
+}
+
+# Add key/value to tests configuration file
+#
+#  $1 - key
+#  $2 - value
+#
+function testscfg_add () {
+  if [[ $# -lt 2 ]]; then
```
Comment (oleorhagen) [inline modules/testscfg.sh:29]: Nit:
-ne 2 ?

File: modules/testscfg.sh [hunk 2]
```diff
@@ -0,0 +1,47 @@
...
+  local -r value="${2}"
+  echo "${key}=\"${value}\"" >> work/testscfg.tmp
+}
+
+# Save tests configuration file into the given location
+#
+#  $1 - Filename
+#
+function testscfg_save () {
+  if [[ $# -lt 1 ]]; then
```
Comment (oleorhagen) [inline modules/testscfg.sh:42]: nit:
-ne 1

File: mender-convert-package [hunk 3]
```diff
@@ -350,35 +351,31 @@ else
...
+testscfg_add "MENDER_PARTITIONING_OVERHEAD_KB" "$(( (${overhead_sectors} * 512) / 1024 ))"
+testscfg_add "MENDER_PARTITION_ALIGNMENT" "${MENDER_PARTITION_ALIGNMENT}"
+testscfg_add "MENDER_STORAGE_TOTAL_SIZE_MB" "${MENDER_STORAGE_TOTAL_SIZE_MB}"
+testscfg_add "MENDER_UBOOT_ENV_STORAGE_DEVICE_OFFSET" "12582912"
+testscfg_add "MENDER_ARTIFACT_NAME" "${artifact_name}"
+testscfg_add "MENDER_FEATURES" "${mender_features}"
+testscfg_add "DEPLOY_DIR_IMAGE" "${PWD}/deploy"
+testscfg_add "MENDER_MACHINE" "${device_type}"
 
 # Outputting device base only relevant for some configurations
```
Comment (oleorhagen) [inline mender-convert-package:371]: Q:
Still only for some configurations?

**Review by oleorhagen — APPROVED** [review]
Nice solution imo

File: mender-convert-package (hunk 3 above)
Comment (lluiscampos) [reply mender-convert-package:371]: A:
Only `if [ "${MENDER_ENABLE_PARTUUID}" != "y" ]`, right? I dunno, it was there and I haven't changed the if condition neither...

**Pull request merged**